src/extract/gadm.py

```diff
@@ -1,13 +1,24 @@
 import os
+import zipfile
 
-from utils.yaml_api import parse_config
+from utils.yaml_api import parse_yaml
 from utils.requests_api import download_url
 
+import pdb
 
-GADM_YEMEN_ADDRESS = 'https://biogeo.ucdavis.edu/data/gadm3.6/gpkg/gadm36_YEM_gpkg.zip'
+
+def get_world():
+    config = parse_yaml('config.yml')
+    rawdir = config['dirs']['raw_data']
+    outputZip = config['surrounding']['gadm']['rawzip']
+    sourceURL = config['surrounding']['gadm']['url']
+    source_gadm_world = os.path.join(rawdir, outputZip)
+    print(r'Downloading {0}'.format(sourceURL))
+    download_url(sourceURL, source_gadm_world)
+    # Unzip - as reading zipped world geopackage takes too long
+    print(r'Unzipping {0}'.format(config['surrounding']['gadm']['url']))
```

Comment:
I think the unzipping should be part of the transformation

Reply:
OK, yes that makes sense.